### PR TITLE
TO DO LIST- CRUD

### PR DESCRIPTION
**Pull request by kensteph**
In this PR I : 
- Create addTodo function.
- Create deleteTodo function.
- Create editTodo function
- Remove fontawsome use of google icon instead
- Rename my js folder to modules and add storage.js, todo-functionalities.js, and todo.js
- Preserved the data in Local Storage
- Remove fontawesome dependencies
- Remove unused code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
+
+const addIcon = document.querySelector('#addIcon');
+addIcon.addEventListener('click', addTodo);
+
+const form = document.querySelector('#addToDoForm');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  addTodo();
+});
+
```
Comment (t-yanick) [inline src/index.js:16]: - [ ] Very good job implementing almost all the functionalities. But there is this requirement I could not find after running your project. 
- `Implement a function for editing task descriptions.`
Kindly get this functionality working as I think it is the only thing left.
image
- As you can see from the screenshot of your project, I can create a ToDO, I can delete, but I cannot edit the description.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @kensteph , 

### Good job so far!

### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (kensteph) [reply src/index.js:16]: Hi @t-yanick , Like the demo. If you click on the todo list text like "test 3". You will be able to edit it.  Here is the real version: https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/. And the Sneak peek: To-do list : http://savefrom.net/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DAcUd-_Yjjqg&utm_source=userjs-chrome&utm_medium=extensions&utm_campaign=link_modifier

Comment (kensteph) [reply src/index.js:16]: Hi @t-yanick ! Are you there?

Comment (t-yanick) [reply src/index.js:16]: Hi @kensteph , You are right. Sorry about missing that. Kindly submit for re-review and your PR will be approved by the reviewer.

Comment (kensteph) [reply src/index.js:16]: Ok @t-yanick . No worries! Thank you

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @kensteph,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Good commit messages 👍🏽
✅ Good descriptive PR 👍🏽
✅ Add, remove, and edit features of a todo work fine as expected 👍🏽
✅ Todos are stored in localStorage with their index arranged as expected 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**